test/test_stories.md

```diff
@@ -4,3 +4,20 @@
 * affirm
     - utter_getstarted
     - utter_first_bot_with_rasa
+
+## tech question, deny
+* technical_question
+  - action_docs_search
+  - utter_ask_docs_help
+  - action_listen
```

Comment:
I don't think test stories should have action_listen in them? or at least don't need them